### PR TITLE
Docs: Symfony Flex installation

### PR DESCRIPTION
**Pull request by renatomefi**
| Q             | A
| ------------- | ---
| Documented?   | yes
| Fixed tickets | #192 
| License       | MIT

- [x] How to install with SF Flex
- [x] Location of Type definition files
- [x] Routes configuration

### CODE REVIEW

**Comment by renatomefi** [comment]
Let's wait before the recipe is merged

File: Resources/doc/index.md [hunk 1]
```diff
@@ -9,7 +9,44 @@ Requirements
...
+
+In the project directory:
+
+```bash
+composer require overblog/graphql-bundle
+```
+
+**b)** Accept the contrib recipes installation from SF Flex
+
+```bash
```
Comment (ooflorent) [inline Resources/doc/index.md]: The `bash` syntax is not relevant here

File: Resources/doc/index.md [hunk 2]
```diff
@@ -9,7 +9,44 @@ Requirements
...
+    The recipe for this package comes from the "contrib" repository, which is open to community contributions.
+    Do you want to execute this recipe?
+    [y] Yes
+    [n] No
+    [a] Yes for all packages, only for the current installation session
+    [p] Yes permanently, never ask again for this project
+    (defaults to n): 
+```
+
+**c)** If you want to run GraphiQL interface twig is require, install it via:
```
Comment (ooflorent) [inline Resources/doc/index.md]: Could you capitalize _Twig_?

File: Resources/doc/index.md [hunk 3]
```diff
@@ -54,7 +91,10 @@ overblog_graphql_graphiql:
     resource: "@OverblogGraphQLBundle/Resources/config/routing/graphiql.yml"
 ```
 
-**e)** Use composer ClassLoader to load generated class (optional but recommended)
+Composer autoloader configuration (optional)
+------------
+
+Use composer ClassLoader to load generated class (optional but recommended)
```
Comment (ooflorent) [inline Resources/doc/index.md]: Could you wrap `ClassLoader` with backticks?

File: Resources/doc/index.md [hunk 4]
```diff
@@ -9,7 +9,44 @@ Requirements
...
+
+**a)** Download the bundle
+
+In the project directory:
+
+```bash
+composer require overblog/graphql-bundle
+```
+
+**b)** Accept the contrib recipes installation from SF Flex
```
Comment (ooflorent) [inline Resources/doc/index.md]:
```diff
- SF
+ Symfony
```

File: Resources/doc/index.md [hunk 5]
```diff
@@ -9,7 +9,44 @@ Requirements
 ------------
 PHP >= 5.5
 
-Installation
+After Installation
```
Comment (ooflorent) [inline Resources/doc/index.md]:
```diff
- Installation
+ installation
```

File: Resources/doc/definitions/schema.md [hunk 6]
```diff
@@ -1,8 +1,25 @@
 Schema
 =======
 
+Default files location
+-------
+
+**Symfony Flex:**
+
+- ***Main configuration:*** ```/config/packages/graphql.yaml```
```
Comment (ooflorent) [inline Resources/doc/definitions/schema.md]: Could you use single backticks?

File: Resources/doc/index.md [hunk 7]
```diff
@@ -9,7 +9,44 @@ Requirements
...
+------------
+
+Define your [graphQL schema](definitions/index.md).
+
+Symfony Flex installation
+------------
+
+**Note:** OverBlogGraphQLBundle only supports Symfony Flex from version 0.9.0 onwards
```
Comment (ooflorent) [inline Resources/doc/index.md]:
```diff
- OverBlogGraphQLBundle
+ OverblogGraphQLBundle
      ^
```

**Comment by renatomefi** [comment]
@ooflorent sure, good tips! thanks :)

**Comment by renatomefi** [comment]
@mcg-web the flex recipe is merged and seems to be working fine, wanna merge this one?

**Comment by mcg-web** [comment]
@renatomefi :+1:  nice

**Pull request merged**